js/components/Feed.jsx

```diff
@@ -61,6 +61,11 @@ var Feed = React.createClass({
 
   render: function() {
     var feedHeader = this.props.userId ? <FeedHeader onClick={this.handleClick}/> : '';
+    var feedText = '';
```

Comment:
Add a brief warning message if there are no feed transactions for a given filter